nfs/src/rpc_task.cpp

```diff
@@ -148,6 +148,15 @@ void rpc_task::init_getattr(fuse_req *request,
     fh_hash = get_client()->get_nfs_inode_from_ino(ino)->get_crc();
 }
 
+void rpc_task::init_statfs(fuse_req *request,
+                           fuse_ino_t ino)
+{
+    assert(get_op_type() == FUSE_STATFS);
+    set_fuse_req(request);
+
+    rpc_api->statfs_task.set_ino(ino);
+}
```

Comment:
following is missing

fh_hash = get_client()->get_nfs_inode_from_ino(ino)->get_crc();
}